debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -2125,6 +2125,15 @@ default String getFallbackStringProperty(Field newProperty, Field oldProperty) {
      * @return the evaluated value
      */
     static String getFallbackStringProperty(Configuration config, Field newProperty, Field oldProperty) {
+        if (null != config.getString(oldProperty.name()) && null != config.getString(newProperty.name())) { // both are set
```

Comment:
```suggestion
        if (config.hasKey(oldProperty.name()) && config.hasKey(newProperty.name())) { // both are set
```

This is a bit more descriptive of what we want to do here. Same in the new method below.

Reply:
Is this the same meaning? `getString()` will retutn a non-null default value, if set for properties that have non-null default values. Agree, the question is what do we want to achieve here?